docs/image.rst

```diff
@@ -0,0 +1,108 @@
+Image
+*****
+
+.. py:module:: microbit
+
+Classes
+=======
+
+.. py:class:: Image(width_or_string=None, height=None, buffer=None)
+
+    If ``width_or_string`` is a string, creates an image from that string.
+
+    If it's an integer, a height has to also be specified, and it creates
+    an image with ``width_or_string`` columns and ``height`` rows. Optionally
+    ``buffer`` can be an array of ``width_or_string``×``height`` integers to
+    initialize the image.
```

Comment:
However, I can see no way to do that with Sphinx, because the py:class is a block. Unless I describe it as two separate classes, but then it's not clear that it's the same class with the same methods (should I duplicate the method descriptions?).

Finally, and I know it's a matter of taste to some degree, this is unpythonic and can lead to surprising results when the parameter you are passing is not the type you expected it to be.